### PR TITLE
increase default memory size for the aarch64 architecture

### PR DESCRIPTION
**Pull request by stlankes**

### CODE REVIEW

File: src/arch/aarch64/mod.rs [hunk 1]
```diff
@@ -70,7 +70,7 @@ pub unsafe fn boot_kernel(
 
 	// Supply the parameters to the HermitCore application.
 	BOOT_INFO.base = virtual_address;
-	BOOT_INFO.limit = RAM_START + 0x10000000 /* 256 MB */;
+	BOOT_INFO.limit = RAM_START + 0x50000000 /* 512 MB */;
```
Comment (mkroening) [inline src/arch/aarch64/mod.rs:73]: The comment does not fit the number.

**Review by mkroening — CHANGES_REQUESTED** [review]
bors r-

File: src/arch/aarch64/mod.rs (hunk 1 above)
Comment (stlankes) [reply src/arch/aarch64/mod.rs:73]: I agree :-)

**Review by mkroening — APPROVED** [review]
bors r+

**Pull request merged**